functionary/ui/forms/tasks.py

```diff
@@ -97,7 +97,7 @@ class TaskParameterForm(Form):
 
     def __init__(
         self,
-        function: "Function",
+        tasked_object: Union["Function", "Workflow"],
```

Comment:
Just an idea: I'm wondering if we should create an abstract base class for models that are able to spawn tasks, so we can avoid needing to maintain all of these `Union`s. We could then simplify our comments/annotations to just specify that the argument must implement that abstract base class. However, if we don't plan on adding anymore models that can create tasks, then it would not be too useful.

Reply:
Yeah, I briefly thought about that and then opted not to for now.  If there was a 3rd item I think I would have gone the other way on it, so maybe I'll make that the threshold.  Or maybe if I find myself doing this Union thing in a bunch more places it'll motivate me to make the base class.